test/end2end/model_test.dart

```diff
@@ -1538,7 +1538,7 @@ void main() {
       expect(
           B.documentationAsHtml,
           contains(
-              '<p>Extends class <a href="${htmlBasePlaceholder}ex/Apple-class.html">Apple</a>, use <a href="${htmlBasePlaceholder}ex/Apple/Apple.html">new Apple</a> or <a href="${htmlBasePlaceholder}ex/Apple/Apple.fromString.html">new Apple.fromString</a></p>'));
+              '<p>Extends class <a href="${htmlBasePlaceholder}ex/Apple-class.html">Apple</a>, use <a href="${htmlBasePlaceholder}ex/Apple/Apple.html">Apple.Apple</a> or <a href="${htmlBasePlaceholder}ex/Apple/Apple.fromString.html">Apple.fromString</a></p>'));
```

Comment:
I think these should be changed to `Apple.new` instead of `Apple.Apple` since `Apple.Apple` is not recognized by the analyzer (and thus the IDE). I don't think we need to test that this syntax works.

Reply:
Ah, good catch, yes this is a better choice.